src/applications/ivc-champva/10-7959f-2/config/form.js

```diff
@@ -244,6 +254,52 @@ const formConfig = {
         },
       },
     },
+    fileUpload: {
+      title: 'Upload files',
+      pages: {
+        page7: {
+          path: 'upload-supporting-documents',
+          title: 'Upload files',
+          CustomPage: FileFieldWrapped,
```

Comment:
Since the `FileFieldWrapped` custom page is mainly a passthrough (`FileUpload.jsx` slaps the `MissingFileOverview` stuff on top of a regular Schema Form) we should be able to remove this prop and it should look the same.